src/components/EditProfile/EditProfile.js

```diff
@@ -12,66 +12,94 @@ import EditPersonalInformationForm from './EditPersonalInformationForm';
     }
     return (
-        <Container component="main" maxWidth="xl" sx={{ pt: "4vh" ,mb: 4 }}>
-            <Box>
-                <Typography component="h1" variant="h4" align="center">
-                    Profile Settings
-                </Typography>   
-                <Box display="flex" justifyContent="flex-end">
-                    <Button 
-                        variant={editingProfile ? "contained" : "outlined"}
-                        onClick={() => {handleEditingProfile()}}>
-                        {editingProfile ? "Save" : "Edit"}
-                    </Button>                            
-                </Box>     
-            </Box>
-            <Paper
-                variant="outlined"
-                sx={paperProps}>
-                <React.Fragment>
-                    <Box sx={{ display: 'flex', justifyContent: 'flex-end' }}>
-                        <EditPersonalInformationForm editing={editingProfile}/>
-                    </Box>
-                </React.Fragment>
-            </Paper>
-            <Grid container spacing={2}>
-                <Grid item xs={12} sm={6}>
-                    <Paper
-                        variant="outlined"
-                        sx={paperProps}>
-                        <React.Fragment>
-                            <Box sx={{ display: 'flex', justifyContent: 'flex-end' }}>
-                                <EditRoommatePrefForm editing={editingProfile}/>
-                            </Box>
-                        </React.Fragment>
-                    </Paper>     
-                </Grid>
-                <Grid item xs={12} sm={6}>
-                    <Paper
-                        variant="outlined"
-                        sx={paperProps}>
-                        <React.Fragment>
-                            <Box sx={{ display: 'flex', justifyContent: 'flex-end' }}>
-                                <EditPropertyPrefForm editing={editingProfile}/> 
-                            </Box>
-                        </React.Fragment>
-                    </Paper>                    
+        <UserInfoContext.Provider  value={{ userInfo, setUserInfo }}>
+            <Container component="main" maxWidth="xl" sx={{ pt: "4vh" ,mb: 4 }}>
+                <Box>
+                    <Typography component="h1" variant="h4" align="center">
+                        Profile Settings
+                    </Typography>   
+                    <Box display="flex" justifyContent="flex-end">
+                        <Button 
+                            variant={editingProfile ? "contained" : "outlined"}
+                            onClick={() => {handleEditingProfile()}}>
+                            {editingProfile ? "Save" : "Edit"}
+                        </Button>                            
+                    </Box>     
+                </Box>
+                <Paper
+                    variant="outlined"
+                    sx={paperProps}>
+                    <React.Fragment>
```

Comment:
I feel like these React.Fragment tags might not be necessary. Because the Fragment tag is a place holder instead of an actual tag element. Could you check if that's true? If so, just remove them.

Reply:
documentation here: <https://beta.reactjs.org/reference/react/Fragment>

Reply:
yeah they weren't doing anything. i removed them 👍